pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -325,6 +327,7 @@ func (ssr *Snapshotter) TakeDeltaSnapshot() error {
 	ssr.prevSnapshot = snap
 	metrics.LatestSnapshotRevision.With(prometheus.Labels{metrics.LabelKind: ssr.prevSnapshot.Kind}).Set(float64(ssr.prevSnapshot.LastRevision))
 	metrics.LatestSnapshotTimestamp.With(prometheus.Labels{metrics.LabelKind: ssr.prevSnapshot.Kind}).Set(float64(ssr.prevSnapshot.CreatedOn.Unix()))
+	metrics.SnapshotRequired.With(prometheus.Labels{metrics.LabelKind: snapstore.SnapshotKindDelta}).Set(0)
```

Comment:
Please add it at https://github.com/gardener/etcd-backup-restore/blob/master/pkg/snapshot/snapshotter/snapshotter.go#L301 as well.

Reply:
Not necessary, because we know that the delta snapshot required will be 0 if there are no new events.